docs/usage/font_installation.rst

```diff
@@ -52,3 +69,18 @@ Make also sure to delete the cache of matplotlib, otherwise the fonts may not be
 
    cache_dir = matplotlib.get_cachedir()
    subprocess.run(["rm", "-rv", cache_dir])
+
+Try to restart your python session and run ``import plothist`` again. If no warning is raised, the fonts are installed correctly and you can skip the rest of this section.
```

Comment:
I would remove this sentence, it is already present at the beginning of the page.